app/factories/bulkrax/object_factory.rb

```diff
@@ -90,17 +90,18 @@ def update
     def find
       found = find_by_id if attributes[:id].present?
       return found if found.present?
-    rescue Valkyrie::Persistence::ObjectNotFoundError
+      return search_by_identifier if attributes[work_identifier].present?
```

Comment:
This change fixed the reason why a link was not being displayed on a successful import. Before it would leave a message that the work didn't successfully import when that wasn't true. 

<img width="668" alt="image" src="https://github.com/samvera/bulkrax/assets/10081604/1cb49813-c2dd-4e83-ab05-34e98eb59f7f">